src/main/java/org/cactoos/text/After.java

```diff
@@ -31,6 +31,9 @@
  * <p>There is no thread-safety guarantee.
  *
  * @since 1.0
+ * @todo #981:30min Update qulice version and remove
```

Comment:
@Iprogrammerr please specify the minimum required version of qulice

Reply:
@victornoel Done.